core/src/it/units/sdm/project/board/gui/CellHighlighter.java

```diff
@@ -0,0 +1,99 @@
+package it.units.sdm.project.board.gui;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.Group;
+import com.badlogic.gdx.scenes.scene2d.ui.Cell;
+import com.badlogic.gdx.scenes.scene2d.ui.Stack;
+import it.units.sdm.project.board.Position;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import static it.units.sdm.project.game.gui.FreedomGame.NUMBER_OF_ROWS;
+
+/**
+ * This class provides helper methods to highlight valid moves in a {@link it.units.sdm.project.game.gui.FreedomGame}.
+ */
+public class CellHighlighter {
+    /**
+     * {@link GuiBoard#DARK_TILE}'s highlighted default {@link Color}
+     */
+    private static final Color HIGHLIGHT_DARK_TILE = new Color(105 / 255f, 105 / 255f, 105 / 255f, 255 / 255f);
```

Comment:
It would be nice if we had a way to obtain this highlighted colors from the board tile colors. (just a dream)